admin-action/shopify.extension.toml.liquid

```diff
@@ -11,20 +11,40 @@ module = "./src/ActionExtension.{{ srcFileExtension }}"
 target = "admin.product-details.action.render"
 
 
-# Valid extension points:
+# Valid extension targets:
+
+# Abandoned checkout details page
+# - admin.abandoned-checkout-details.action.render
 
 # Customer index and detail pages
 # - admin.customer-index.action.render
 # - admin.customer-index.selection-action.render
 # - admin.customer-details.action.render
 # - admin.customer-segment-details.action.render
 
-# Order index and detail pages
+# Discount index and detail pages
+# - admin.discount-index.action.render
+# - admin.discount-details.action.render
+
+# Draft order index and detail pages
+# - admin.draft-order-details.action.render
+# - admin.draft-order-index.action.render
+# - admin.draft-order-index.selection-action.render
+
+# Collection index and detail pages
+# - admin.collection-details.action.render
+# - admin.collection-index.action.render
+
+# Order index, detail pages and order fulfilled card
 # - admin.order-index.action.render
 # - admin.order-index.selection-action.render
 # - admin.order-details.action.render
+# - admin.order-fulfilled-card.action.render
```

Comment:
wasn't sure if this should be in it's own category since it's kind of its own thing despite being located so closely to the others